tv_apps/smart-home/test/marionette/smart_home_test.js

```diff
@@ -50,6 +50,17 @@ marionette('Smart Home', function() {
       var editButton = client.findElement('#edit-button');
       var element = client.findElement('#main-section');
       var searchButton = client.findElement('#search-button');
+      var deckTvCard = client.findElement('#card-list > .card');
+      var deckTvButton = client.findElement('.app-button.deck-tv');
```

Comment:
Hi
These 2 elements all have been set with the 'focused' class when getting focus
Thanks